src/Learning/KWDataPreparation/KWClassStats.cpp

```diff
@@ -857,8 +861,9 @@ void KWClassStats::WriteReport(ostream& ost)
 	ost << "Selection variable\t" << TSV::Export(GetDatabase()->GetSelectionAttribute()) << "\n";
 	ost << "Selection value\t" << TSV::Export(GetDatabase()->GetSelectionValue()) << "\n";
 
-	// Nombre d'instances
-	ost << "Instances\t" << GetInstanceNumber() << "\n";
+	// Nombre d'instances si disponible
+	if (GetLearningSpec()->IsTargetStatsComputed())
+		ost << "Instances\t" << GetInstanceNumber() << "\n";
```

Comment:
Could we have `*GetLearningSpec().isTargetStatsComputed()` equal to `true` and `GetInstanceNumber()` equal to `0`?

Reply:
Oui:
- IsTargetStatsComputed indique que l'on a réussi avec succès à analyser la base
- et dans ce cas, le résulta de l'analyse peut impliquer GetInstanceNumber() == 0
  - par exemple si fichier vide, ou si le critère de sélection ne sélectionne aucune instance

Reply:
Fait